Lib/test/test_move.py

```diff
@@ -0,0 +1,282 @@
+import unittest
```

Comment:
Minor point and I'm not clear: would it make sense to have a test using `__slots__` to restrict the fields to document that that works?

Reply:
Good call, I added a test for it